frontend/src/app/App.tsx

```diff
@@ -118,100 +119,120 @@ const App = () => {
     <>
       <VersionEnforcer />
-      <MaintenanceBanner />
+      {process.env.REACT_APP_DISABLE_MAINTENANCE_BANNER === "true" ? null : (
+        <MaintenanceBanner />
+      )}
       {process.env.REACT_APP_IS_TRAINING_SITE === "true" && (
         <TrainingNotification />
       )}
       <WithFacility>
         <Page>
           <Header />
-          <Switch>
-            <Route
-              path="/queue"
-              render={() => {
-                return <TestQueueContainer />;
-              }}
-            />
+          <Routes>
             <Route
               path="/"
-              exact
-              render={({ location }) => (
-                <Redirect
-                  to={{
-                    ...location,
-                    pathname: homepagePath,
-                  }}
+              element={
+                <Navigate
+                  to={{ pathname: homepagePath, search: location.search }}
                 />
-              )}
+              }
             />
-            <ProtectedRoute
-              path="/results/:page"
-              render={({ match }: any) => {
-                return <TestResultsList pageNumber={match.params.page} />;
-              }}
-              requiredPermissions={appPermissions.results.canView}
-              userPermissions={data.whoami.permissions}
+            <Route path="queue" element={<TestQueueContainer />} />
+            <Route
+              path="results/:pageNumber"
+              element={
+                <ProtectedRoute
+                  requiredPermissions={appPermissions.results.canView}
+                  userPermissions={data.whoami.permissions}
+                  element={<TestResultsList />}
+                />
+              }
             />
-            <ProtectedRoute
-              path="/results/"
-              render={() => <CleanTestResultsList />}
-              requiredPermissions={appPermissions.results.canView}
-              userPermissions={data.whoami.permissions}
+            <Route
+              path="results"
+              element={
+                <ProtectedRoute
+                  requiredPermissions={appPermissions.results.canView}
+                  userPermissions={data.whoami.permissions}
+                  element={<CleanTestResultsList />}
+                />
+              }
             />
-            <ProtectedRoute
-              path={`/patients/:page?`}
-              render={({ match }: any) => {
-                return <ManagePatientsContainer page={match.params.page} />;
-              }}
-              requiredPermissions={appPermissions.people.canView}
-              userPermissions={data.whoami.permissions}
+            <Route
+              path="patients/:pageNumber"
+              element={
+                <ProtectedRoute
+                  requiredPermissions={appPermissions.people.canView}
+                  userPermissions={data.whoami.permissions}
+                  element={<ManagePatientsContainer />}
```

Comment:
Yeah not a bad idea! The reason I had to duplicate this route is because v5 supported optional URL params like `/patients/:pageNumber?` which would match both `/patients` and `/patients/1`, but v6 no longer supports that. Hence the need for two routes. But I will definitely DRY it up

Reply:
Soooo I tried to do this, and for some reason it broke the patients page. It kept throwing "The operation is insecure", and I couldn't figure out why :thinking: so I reverted for now.